samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -79,6 +79,18 @@ Recovery mode
 In the provisioned state, you can enter the recovery mode for a limited time.
 This mode allows the Android device to recover a lost provisioning key from the locator tag.
 
+DULT Motion detector active mode
```

Comment:
We can call it simply as we did for identification mode that is also DULT-based:

```suggestion
Motion detection mode
```

Reply:
I'd rather leave it as is. It's easier to later say that "motion detector is active" rather than saying "accessory is in motion detection mode". Also it will be better aligned with the logs on device. See LOG_INF("FMDN: motion detector started");, LOG_INF("FMDN: motion detector stopped");, LOG_INF("FMDN: motion indicated: motion detector is inactive");.

Reply:
you can still say "motion detector is active", this is more about making the title shorter and less packed with information.

```
In the provisioned state, the accessory can activate the motion detector mode from the DULT specification.
```